components/raftstore/src/store/peer.rs

```diff
@@ -3509,7 +3571,8 @@ where
             || self.has_pending_snapshot()
             || msg.get_from() != self.leader_id()
             // For followers whose disk is full.
-            || !matches!(ctx.self_disk_usage, DiskUsage::Normal)
+            || (!matches!(ctx.self_disk_usage, DiskUsage::Normal) &&
+            matches!(peer_disk_usage,DiskUsage::Normal))
```

Comment:
1. Try not to transfer leader to a disk full node to avoid write failure.
2. But if the currently Leader is disk full, then the target node should accept it event it is disk full.

Reply:
Done